### ISSUE TITLE
AllowDBNull column from table schema obtained from DataReader is not set

### ISSUE THREAD

**Issue by weronikaejsmont** [Cinchoo/ChoETL#38]
Setting IsNullable property of ChoCSVRecordFieldConfiguration object doesn't fill AllowDBNull column values from table schema (dataReader.GetSchemaTable()) of dataReader obtained from AsDataReader() method.

Is it possible to pass information about nullable columns to datareader?

**Comment by Cinchoo** [comment]
Can you please add more information with samples?

**Comment by weronikaejsmont** [comment]
We create some config file (ChoCSVRecordConfiguration) that is further passed to ChoCSVReader object.
The config looks like that:
```
            var config = new ChoCSVRecordConfiguration
            {
                FileHeaderConfiguration = { HasHeaderRecord = true },
                ThrowAndStopOnMissingField = false,
                Delimiter = ",",
                CultureName = CultureInfo.InvariantCulture.ToString()
            };

          var row = new ChoCSVRecordFieldConfiguration(name, position)
                {
                    AltFieldNames = id,
                    Size = 1000,
                    FieldType = type,
                    Truncate = false,
                    NullValue = "",
                    IsNullable = false,
                    ErrorMode = ChoErrorMode.ThrowAndStop,
                    ValueConverter = s => someParser
                };
config.CSVRecordFieldConfigurations.Add(row);
```
from created ChoCSVReader instance we run AsDataReader() method to get datareader.

After that we get table schema (dataReader.GetSchemaTable()). In schema are stored information about columns. For example AllowDBNull property (dataReader.GetSchemaTable().Rows[0]["AllowsDBNull"]). This property isn't set. But it should be set accordingly to IsNullable value per column.

**Comment by Cinchoo** [comment]
Put a fix, pushed new version 1.0.9.1 to nuget. try it.

You will have to set the fieldType to nullable type. 

respecting 'IsNullable' will take some time to put a fix.
